src/main/plugins/org.dita.pdf2/resource/messages.xml

```diff
@@ -60,11 +60,11 @@ See the accompanying LICENSE file for applicable license.
     <reason>There are multiple index entries found to close the index range for "%1".</reason>
     <response>Ensure that any index term with start="%1" has only one matching end term with end="%1".</response>
   </message>
-  <message id="PDFX004F" type="FATAL">
+  <message id="PDFX004F" type="ERROR">
```

Comment:
At least once before in cases like this I've created a new message, so in this case something like `PDFX999E`, so that the message ID and the message type will match. Though, I'm ok with them not matching.